src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:25321/organizations/org2/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:25321/project/issues?id=sample&pullRequest=4998&issues=AWHDa4cC7pweg8JCD_97&open=AWHDa4cC7pweg8JCD_97)

[ISSUE_KEY]: AWHDa4cC7pweg8JCD_97